### DISCUSSION TITLE
Why does Microsoft Teams cause GeekBench 6 score to increase?

### DISCUSSION THREAD

**Discussion by pulpinex** [ChefKissInc/NootRX#25] [Q&A] [answered]
Hi everyone,
today, by chance, I made a strange "discovery"...
I tested the dGPU (an RX 6900 XT) on which I am testing the NootRx kext on my hackintosh... (Yes, I know that it is supported natively and NootRx is not needed, but I can't stay without experimenting).

Surprisingly, while using Teams (in call, not only opened) I noticed an extraordinary jump in the score.
Let me explain better: Geekbench 6 with Wathewergreen... Metal score around 217,000, with NootRx Metal score around 223,000....

Today I did the exact same test while using Teams (I was doing a call); so I did the geekbench test... by chance, just to play around.
Boooommmm!
Metal result (with Nootrx): 262.000!
OpneCL (with Nootrx): 128/129.000

Closing the call the score returns to normal value (127/128K in OpenCL, 227/228K in Metal).
So only doing a call the score jump to a 15% up.
Can anyone explain why?
Is this a geekbench bug or something else?

**Comment by VisualEhrmanntraut** [comment, marked as answer]
Microsoft Teams might be disabling V-sync or something like that.

**Reply by pulpinex** [reply]
thx for your reply. :)
